### ISSUE TITLE
Acrylic effect on Windows 10 causes performance issues, while not working at all

### ISSUE THREAD

**Issue by fishchev** [DevToys-app/DevToys#1258]
### Current behavior

Acrylic effect on Windows 10 doesn't seem to work at all _and_ while not working causes performance issues when window 
is moved or resized with high-pooling rate mouse.

### How to reproduce it (as minimally and precisely as possible)

Start GUI on W10 (19045.4529), observe lack of acrylic.
Then move/resize the window with mouse that has high pooling rate (observable with 250Hz and up on my system), window would be severely lagging behind your mouse input.

**Note** that for some reason performance side of the problem goes away if VS debugger with enabled `XAML Hot Reload` is attached. So disable it if you need to debug: `Tools > Options > Debugging > XAML Hot Reload > Enable XAML Hot Reload`.

### Expected behavior

Fixing both would be cool, but I personally don't care for acrylic, so would be happy just to not have window lag behind cursor.

### Screenshots

1. Settings app with working acrylic
2. Custom build where AccentState is not set
3. v2.0.2.0 release

https://github.com/DevToys-app/DevToys/assets/3948703/96940d23-6b49-4965-969c-99a57516990a

Yea, you saw a second cursor with current release in the end -- all window manipulation events get queued up and executed with phantom cursor long after I released mouse button

### Workaround

As currently acrylic effect doesn't seem to work at all you could just not set
`AccentState = AccentState.ACCENT_ENABLE_ACRYLICBLURBEHIND,` 
in [MicaWindowWithOverlay.xaml.cs:406](https://github.com/DevToys-app/DevToys/blob/6f22ed6cd72269583a643638404d18df31ecbeb4/src/app/dev/platforms/desktop/DevToys.Windows/Controls/MicaWindowWithOverlay.xaml.cs#L406) and moving/resizing performance issue goes away.

That leads to background changing from solid neutral gray to pure white, but that seems like an easy thing to fix (probably not even a problem as you could see on the video above)

### Affected platforms

Windows

### Affected DevToys kind

DevToys (app with GUI)

### DevToys Version

tag: v2.0.2.0

**Comment by veler** [comment]
Hi,
Thanks for sharing this feedback. I'm sorry but I'm not perceiving the lag on the video you shared. Do you have a high refresh monitor perhaps? Mine is 60Hz.

**Comment by fishchev** [comment]
> Do you have a high refresh monitor perhaps? Mine is 60Hz.

Yes, but video is only 30 fps, so shouldn't be a problem. 

> I'm sorry but I'm not perceiving the lag on the video you shared.

Look how fast movement happens and how sporadically velocity changes of first two windows, that's how I move the mouse.
Then compare how third window behaves, while I move the mouse in the very same fashion - window pans really slow in comparison and does so at almost constant speed.

My guess is that each movement event causes a redraw or some other blocking call, which executes at more or less consistent speed, thus making window move at more or less constant speed of 1px per update.

Maybe it's a bit hard to feel on that video if you're not the one that recorded it 😁
If that's the case, _then there's way more obvious red flag:_

> Yea, you saw a second cursor with current release in the end -- all window manipulation events get queued up and executed with phantom cursor long after I released mouse button

Window continues to moves for whole second (!) after I've released the mouse button and stopped any movement, second cursor that's static is the real one. E.g. _look at 0:32_, see clicking hand cursor popping in? That's the real mouse position, I'm not moving mouse anymore and just waiting for it to processes all movements I did with the window.
